content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,431 @@
+   upper right corner of the page, click the :guilabel:`Share` button, and slide the
+   :guilabel:`Share to Web` toggle switch until it reads :guilabel:`Article Published`. It can then
+   be added to a *Helpdesk* team.
+
+Once an article has been created and assigned to a *Helpdesk* team, content can be added and
+organized through the *Knowledge* app.
+
+.. seealso::
+   :doc:`Editing Knowledge articles </applications/productivity/knowledge/articles_editing>`
+
+Search articles from a Helpdesk ticket
+--------------------------------------
+
+When members of a *Helpdesk* team are trying to solve a ticket, they can search through the content
+in the *Knowledge* app for more information on the issue.
+
+To search *Knowledge* articles, open a :guilabel:`ticket` - either from the *Helpdesk* app
+dashboard, or by going to :menuselection:`Helpdesk app --> Tickets --> All Tickets`, and selecting a
+:guilabel:`Ticket` from the list.
+
+When a :guilabel:`Ticket` is selected, Odoo reveals that ticket's detail form.
+
+Click the :guilabel:`Knowledge (book)` icon, located above the chatter to open a search window.
+
+.. image:: help_center/help-center-knowledge-search.png
+   :align: center
+   :alt: View of knowledge search window from a helpdesk ticket.
+
+.. tip::
+   *Knowledge* articles can also be searched by pressing :command:`Ctrl + K` to open the command
+   palette, then typing :command:`?`, followed by the name of the desired article.
+
+When Odoo reveals the desired article, click it, or highlight the :guilabel:`Article` title, and
+press :command:`Enter`. This will open the article in the :guilabel:`Knowledge` application.
+
+To open the article in a new tab, press :command:`Ctrl + Enter`.
+
+.. tip::
+   If a more in-depth search is required, press :command:`Alt + B`. That will reveal a separate
+   page, in which a more detailed search can occur.
+
+Share articles to the help center
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order for a *Knowledge* article to be available to customers and website visitors, it has to be
+published.
+
+.. note::
+   Even though the *Help* article has been enabled on a team, Odoo will not share all the nested
+   articles to the web. Individual articles intended for customers *must* be published for them to
+   be viewable on the website.
+
+To publish an article, navigate to the desired article, (by following the above steps), and click
+:guilabel:`Share` icon in the upper-right corner. This will reveal a menu. Slide the toggle button
+labeled :guilabel:`Share to Web` to read :guilabel:`Article Published`.
+
+.. image:: help_center/help-center-knowledge-sharing.png
+   :align: center
+   :alt: View of a knowledge article focused on sharing and publishing options.
+
+Solve tickets with templates
+----------------------------
+
+:guilabel:`Template` boxes can be added to *Knowledge* articles to allow content to be reused,
```

Comment:
why gui here?

double-check all of the guis in this **Add templates to articles** section...are they needed or justified?

Specifically, are there fields or labels named **Template** or **Templates**?

In the following section, the words "template" and "template" are not capitalized and don't have guis...

```suggestion
*Template* boxes can be added to *Knowledge* articles to allow content to be reused,
```